test-d/index.d-test.ts

```diff
@@ -31,6 +31,10 @@ const serverWithModuleAndAllOptions = lc39(plugin, {
   prefix: '/prefix',
   stream: logStream,
   redact: ['authorization'],
+  customLevels: {
```

Comment:
Also use the custom level inside the route, it is correctly typing?

Reply:
Actually there is a problem:

if I remove the `| LevelWithSilentOrString` from 
`[level: string] : LogFn | LevelWithSilentOrString` it is correctly typing, but there is this error running the tests:types:
` ✖  57:4  Property level of type LevelWithSilentOrString is not assignable to string index type LogFn.`

I don't really know how to fix it and I'm stuck. I think we have to use string as type of the key, otherwise is not possible to allow a custom name for log levels. I'm asking for help.